tests/sonic_xcvr/test_cmis.py

```diff
@@ -1266,9 +1266,40 @@ def test_get_transceiver_info(self, mock_response, expected):
                 'N/A',
                 50, 3.3,
                 ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
+                None,
+                None,
+                False, False, False, False, False, False,
+                {'monitor value': 40},
+                None,
+            ],
+            {
+                'temperature': 50,
+                'voltage': 3.3,
+                'tx1power': 'N/A', 'tx2power': 'N/A', 'tx3power': 'N/A', 'tx4power': 'N/A',
+                'tx5power': 'N/A', 'tx6power': 'N/A', 'tx7power': 'N/A', 'tx8power': 'N/A',
+                'rx1power': 'N/A', 'rx2power': 'N/A', 'rx3power': 'N/A', 'rx4power': 'N/A',
+                'rx5power': 'N/A', 'rx6power': 'N/A', 'rx7power': 'N/A', 'rx8power': 'N/A',
+                'tx1bias': 'N/A', 'tx2bias': 'N/A', 'tx3bias': 'N/A', 'tx4bias': 'N/A',
+                'tx5bias': 'N/A', 'tx6bias': 'N/A', 'tx7bias': 'N/A', 'tx8bias': 'N/A',
+                'rx_los': 'N/A',
+                'tx_fault': 'N/A',
+                'tx1disable': 'N/A', 'tx2disable': 'N/A', 'tx3disable': 'N/A', 'tx4disable': 'N/A',
+                'tx5disable': 'N/A', 'tx6disable': 'N/A', 'tx7disable': 'N/A', 'tx8disable': 'N/A',
+                'tx_disabled_channel': 'N/A',
+                'laser_temperature': 40
+            }
+        ),
+        (
+            [
                 ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
                 ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
-                False, False, False, False, False, False,
+                ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
```

Comment:
Please add a test case where there is aread error and tx_power and rx_power returned is None

Reply:
Is this what you want?
```
            [
                ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
                ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
                ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
                'N/A',
                50, 3.3,
                ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'],
                None, =========== tx_power
                None, =========== rx_power
                False, False, False, False, False, False,
                {'monitor value': 40},
                None,
            ],
```
https://github.com/sonic-net/sonic-platform-common/pull/351/files/fb813acb2cd07143c74dfc63717b6a8955deb3c1#diff-c4c26365d2facd90024b8631863c6d9756ebede9a3fd781a0a47848901c37096R1269